main.py

```diff
@@ -15,10 +16,19 @@
     db_pass = row['password']
     db_sid = row['sid']
 
-    conn = ut.oracle_connect(db_ip, db_user, db_pass, db_sid)
-    sql = ut.read_query(sql_path)
-    site_results = ut.collect_data(sql, conn, site)
-    results = results.append(site_results)
+    ut.logging.info('Start Working on ' + site.upper())
+
+    try:
+        conn = ut.oracle_connect(db_ip, db_user, db_pass, db_sid)
+        sql = ut.read_query(sql_path)
+        site_results = ut.collect_data(sql, conn, site)
+        results = results.append(site_results)
+        ut.logging.info('Finish working on ' + site.upper())
+    except Exception as e:
+        ut.logging.error(str(e) + 'was raise due to the running of the main script')

```

Comment:
try to make the error text more clear - for example in site X there was exception raises as : exception